test/gtest/uct/test_ud_pending.cc

```diff
@@ -43,6 +43,33 @@ class test_ud_pending : public ud_base_test {
         return UCS_ERR_BUSY;
     }
 
+    uct_pending_req_t r[64];
```

Comment:
no. requests are posted in post_pending_reqs() and are completed during progress or flush. So i can not have them on stack.